### PR TITLE
configuration update to v1.7.8

### PR DESCRIPTION
**Pull request by haouc**
**What type of PR is this?**
Configuration updates for v1.7.8 release

**Which issue does this PR fix**:
New release

**What does this PR do / Why do we need it**:
Update configuration files

**If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades. Has updating a running cluster been tested?**:

**Does this change require updates to the CNI daemonset config files to work?**:

No
**Does this PR introduce any user-facing change?**:

Yes
```release-note
https://github.com/aws/amazon-vpc-cni-k8s/releases
```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: config/v1.7/calico.yaml [hunk 1]
```diff
@@ -32,7 +32,12 @@ spec:
         # container programs network policy and routes on each
         # host.
         - name: calico-node
-          image: quay.io/calico/node:v3.15.1
```
Comment (jayanthvn) [inline config/v1.7/calico.yaml:35]: Why do we have calico config changes?

Comment (haouc) [reply config/v1.7/calico.yaml:35]: We should have calico copied from master to v1.7. Seems we didn't do that in previous release?

Comment (jayanthvn) [reply config/v1.7/calico.yaml]: Master/calico has some changes which needs to be merged to release-1.7. Changes will be merge to release-1.7 in the next release.

**Review by jayanthvn — APPROVED** [review]
LGTM :)

**Pull request merged**